src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -54,6 +54,7 @@
     <ExpectedDependency Include="System.Memory"/>
     <!-- We do not have a training scenario that covers lib/netstandard2.0/System.Text.Encoding.CodePages.dll  -->
     <ExpectedDependency Include="System.Text.Encoding.CodePages" />
+    <ExpectedDependency Include="System.Threading.Channels" />
```

Comment:
Please add this change back https://github.com/dotnet/roslyn/pull/65028

I have a PR on VS side that transfers the ownership of "System.Threading.Channels" so we will not need this in the vsix
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS/pullrequest/432121

Reply:
will do!

Reply:
done!